src/RemoteRequest/CachedRemoteGetRequest.php

```diff
@@ -101,42 +103,48 @@ public function __construct(
 	 *
 	 * @param string $url URL to get.
 	 * @return Response Response for the executed request.
-	 * @throws FailedToGetFromRemoteUrl If retrieving the contents from the URL failed.
+	 * @throws FailedRemoteRequest If retrieving the contents from the URL failed.
```

Comment:
This change is causing a PHPCS error. It seems it does need to be:

```suggestion
	 * @throws FailedToGetCachedResponse If retrieving the contents from the URL failed.
```